src/fileformats/ocd_file_import.cpp

```diff
@@ -360,7 +360,7 @@ void OcdFileImport::applyGridAndZone(Georeferencing& georef, const QString& comb
 		values.reserve(1);
 		values.push_back(QLatin1String{"3067"});
 	}
-	else if (combined_grid_zone == QLatin1String("14000"))
+	else if (combined_grid_zone == QLatin1String("14001"))
 	{
```

Comment:
Regarding the reference format, IMHO the unqualified number in "Fixes" creates platform lock-in. In case the project migrates to a different hosting platform, it would become unclear what the number is referring to. I'm not disputing the practice (yet), merely offering an different angle of view.